config/routes.rb

```diff
@@ -153,7 +153,10 @@
         only: %i[create destroy],
         constraints: { format: :js },
         defaults: { format: :js }
-      resources :versions, only: %i[show index]
+      resources :versions, only: %i[show index] do
+        get '/dependencies', to: 'versions#dependencies_show'
```

Comment:
I think we can keep controllers small and keep them "RESTful" -> create new controller for dependencies and also for transitive dependencies having only show action.

Reply:
Hi, Thanks. I've changed it.